doc/cprover-manual/contracts-quantifiers.md

```diff
@@ -0,0 +1,43 @@
+# Quantifiers
+
+### Syntax
+
+```c
+__CPROVER_forall { *identifier* *type*; *boolean expression* }
+__CPROVER_exists { *identifier* *type*; *boolean expression* }
```

Comment:
I think these two lines should have `*type* *identifier*`, like `int i`, and not `*identifier* *type*`.

Reply:
Fixed it.